src/OmniSharp.Host/Program.cs

```diff
@@ -100,6 +106,8 @@ public static void Main(string[] args)
 
             if (transportType == TransportType.Stdio)
             {
+                Console.InputEncoding = encoding;
+                Console.OutputEncoding = encoding;
```

Comment:
Check that encoding is not null before setting InputEncoding and OutputEncoding:

```C#
if (encoding != null)
{
    Console.InputEncoding = encoding;
    Console.OutputEncoding = encoding;
}
```